jvector-examples/src/main/java/io/github/jbellis/jvector/example/SiftSmall.java

```diff
@@ -101,7 +105,15 @@ private static void testRecallInternal(GraphIndex graph, RandomAccessVectorValue
             }
             else {
                 NodeSimilarity.ApproximateScoreFunction sf = compressedVectors.approximateScoreFunctionFor(queryVector, VectorSimilarityFunction.EUCLIDEAN);
-                NodeSimilarity.Reranker rr = (j) -> VectorSimilarityFunction.EUCLIDEAN.compare(queryVector, view.getVector(j));
+                NodeSimilarity.Reranker rr = (nodes, results) -> {
+                    var nodeCount = nodes.length;
```

Comment:
this is kind of a lot to expect the average user to implement, can we make it reusable somehow?